### ISSUE TITLE
Error during installation (pip install -r OE2_Requirements.txt)

### ISSUE THREAD

**Issue by the4ndy** [qgriffith/OpenEats#18]
I spun up a fresh Ubuntu 14.04 VM and began following the installation instructions and so far so good until I attempt to install the requirements via the command `pip install -r OE2_Requirements.txt`  

The general error I get is 
 compilation terminated.
    error: command 'x86_64-linux-gnu-gcc' failed with exit status 1

The full paste of the output is here - http://pastebin.com/jBy4nDMr

It looks like there are a few instances of that 'x86_64-linux-gnu-gcc' failing with exit status 1. I did a search on just that error and while it wasnt related to OpenEats, the forum suggested I try installing just a plethora of packages (it seems like a throw a bunch and see what sticks attitude but I didnt see what could hurt, I can pair down later if need be). That did not seem to help much. The specific packages i installed were from here http://stackoverflow.com/a/26333011

I am guessing I overlooked something simple, but I kind of in a rut and was hoping someone could assist, in the meantime though I will keep trying :smile:

**Comment by john-clark** [comment]
you have to have several dev libraries installed.

you also have to be in the correct python environment (basically bash variables that set it up so you can use old versions)

**Comment by the4ndy** [comment]
yeah i was thinking that, but am not nearly qualified to comment on it.  I have attached a link to the pip.log with all the errors on a clean install that follows the instructions to the T

https://gist.github.com/the4ndy/2a87a25e3991137feb74ee949be34e7b

**Comment by goudarziha** [comment]
I had this same issue, in case you haven't solved it yet try 
`sudo apt-get install libxml12-dev libxslt1-dev`

**Comment by mrtumnus** [comment]
This is still broken for me.  The package that the install breaks on is lxml.  I worked around it by installing the current version of lxml (4.2.5) using:
`pip install lxml`
I then commented out the lxml requirement from OE2_Requirements.txt, and it proceeded to succeed.
